promql/functions.go

```diff
@@ -1417,6 +1423,11 @@ func copyParserFunction(fromName, toName string) {
 }
 */
 
+func repointParserFunctions(name, newName string) {
+	parser.Functions[name] = parser.Functions[newName]
+	parser.Functions[name].Name = newName
```

Comment:
I don't think this second line should be necessary, since on the previous line, `parser.Functions[newName].Name` would already have been equal to `NewName`. I think I'll add a TODO comment.

Reply:
And also, the code winds up in this state, which seems a bit error-prone to me:
```
var Functions = map[string]*Function{
...
	"yincrease": {
		Name:       "yincrease",
		ArgTypes:   []ValueType{ValueTypeMatrix},
		ReturnType: ValueTypeVector,
		ExtRange:   true,
	},
	"increase": <pointer to same Function above>
...
```
So the function at ["increase"] is pointing to a Function struct that says its name is `yincrease`. I don't know when they use that redundant Name, but if they ever do, it won't match what they started with ("increase").